build fail is
```
======================================================================
FAIL: test_update (corehq.apps.api.tests.TestGroupResource)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/api/tests.py", line 1198, in test_update
    self.assertEqual(modified.name, "test group")
AssertionError: u'test' != 'test group'
----------------------------------------------------------------------
```
seems related?

--- a/corehq/apps/domain/dbaccessors.py
+++ b/corehq/apps/domain/dbaccessors.py
@@ -33,6 +33,7 @@ def get_docs_in_domain_by_class(domain, doc_class):
         'CallCenterIndicatorConfig',
         'CommCareCaseGroup',
         'CommtrackConfig',
+        'Group',
         'Invitation',
         'PerformanceConfiguration',
         'ReportConfiguration',

--- a/corehq/apps/groups/models.py
+++ b/corehq/apps/groups/models.py
@@ -8,8 +8,9 @@ from corehq.apps.users.models import CouchUser, CommCareUser
 from dimagi.utils.couch.undo import UndoableDocument, DeleteDocRecord, DELETED_SUFFIX
 from datetime import datetime
 from corehq.apps.groups.dbaccessors import (
+    get_group_ids_by_domain,
+    group_by_domain,
     refresh_group_views,
-    stale_group_by_domain,
     stale_group_by_name,
 )
 from corehq.apps.locations.models import SQLLocation
@@ -164,7 +165,7 @@ class Group(UndoableDocument):
 
     @classmethod
     def by_domain(cls, domain):
-        return stale_group_by_domain(domain)
+        return group_by_domain(domain)
 
     @classmethod
     def choices_by_domain(cls, domain):
@@ -176,7 +177,7 @@ class Group(UndoableDocument):
 
     @classmethod
     def ids_by_domain(cls, domain):
-        return [r['id'] for r in stale_group_by_domain(domain, include_docs=False)]
+        return get_group_ids_by_domain(domain)
 
     @classmethod
     def by_name(cls, domain, name, one=True):

--- a/corehq/apps/sms/views.py
+++ b/corehq/apps/sms/views.py
@@ -213,7 +213,7 @@ def get_sms_autocomplete_context(request, domain):
     phone_users = CouchUser.view("users/phone_users_by_domain",
         startkey=[domain], endkey=[domain, {}], include_docs=True
     )
-    groups = Group.view("groups/by_domain", key=domain, include_docs=True)
+    groups = Group.by_domain(domain)
 
     contacts = ["[send to all]"]
     contacts.extend(['%s [group]' % group.name for group in groups])

